mappings/case-list-full.json

```diff
@@ -281,6 +279,11 @@
           ],
           "defendantSex": "M",
           "defendantName": "MR Webb Mitchell",
+          "name": {
+            "title": "Mister",
+           "forename1": "webster",
```

Comment:
We could probably do with an example in the mock data where there is no `forename1` or `surname` just to show that it defaults to `defendantName`.

Reply:
Modification has been done to `MISS FELICIA Villarreality`